ttnn/cpp/ttnn/operations/pool/maxpool/max_pool.hpp

```diff
@@ -12,13 +12,13 @@
 #include "ttnn/deprecated/tt_dnn/op_library/sliding_window_op_infra/sliding_window.hpp"
 
 
+namespace tt {
+namespace tt_metal {
+
 inline uint32_t ceil_multiple_of(uint32_t n, uint32_t m) {
```

Comment:
same function is declared in ttnn/cpp/ttnn/operations/pool/maxpool/device/max_pool2d_device_op.hpp

Reply:
Ah yes, this file will be deleted next :)